### DISCUSSION TITLE
Easy solution for fedora toolbox

### DISCUSSION THREAD

**Discussion by benjamin051000** [kovidgoyal/kitty#5965] [General]
I know it's documented pretty well, but if anyone else has this issue maybe it'll help them. 

I was unable to get kitty's `clear` and `^L` working in Fedora toolbox. The solution is, once inside the toolbox, run `sudo dnf install kitty-terminfo`. 

It seems like this works well for containers in general, as per other discussion threads.

**Comment by page-down** [comment]
One thing to keep in mind is the version of `kitty-terminfo`, if it is too old and does not match the current kitty, there may be problems.
For example, use the latest kitty to connect to a host that has `kitty-terminfo` 0.19.x installed.

The most recent update is the following PR.
https://github.com/kovidgoyal/kitty/pull/5970

kitty version 0.27.0 requires terminfo for 0.27.1 to match precisely.
